kernel/sql-federation/core/src/main/java/org/apache/shardingsphere/sqlfederation/compiler/metadata/schema/SQLFederationTable.java

```diff
@@ -124,4 +133,16 @@ public String toString() {
     public Statistic getStatistic() {
         return statistic;
     }
+    
+    @Override
+    public Collection<Object[]> getModifiableCollection() {
+        return new LinkedList<>();
```

Comment:
Why return empty list here?

Reply:
If the implementation of the getModifiableCollection method returns null, it will trigger a NullPointerException for the EnumerableTableModify class.  However, for our implemented class EnumerableModify, it is allowed to return null and it should not call the getModifiableCollection method in its future implementations.

Reply:
I have changed the return type of this method.